UI/styles.css

```diff
@@ -0,0 +1,387 @@
+    text-align: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    font-weight: bold;
+}
+
+.calendar_block {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 85px;
+    height: 65px;
+    font-family: Epilogue;
+    font-size: 30px;
+    color: #9fb1fe;
+    font-weight: bold;
+}
+
+.calendar {
+    color: #e1e6ff;
+    font-family: Epilogue;
+    font-size: 15px;
+    border: 1px solid #9fb1fe;
+    margin-bottom: 20px;
+    text-align: center;
+    font-weight: normal;
+}
+
+.select_vendor_button {
+    padding: 25px 80px;
+    color: #e1e6ff;
+    font-family: Epilogue;
+    font-size: 20px;
+    border: 1px solid #9fb1fe;
+    margin-left: 70px;
+    margin-top: 10px;
+}
+
+.hashtags_blocks {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 30px;
+    color: #9fb1fe;
+    font-family: Epilogue;
+    font-size: 30px;
+    line-height: 1.5;
+}
+
+/* Main */
+.row {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.left_add {
+    display: block;
+    width: 50%;
+    border-right: 2px solid #9fb1fe;
+}
+
+.right_add {
+    display: block;
+    width: 50%;
+}
+
+.ad_title {
+    font-family: Epilogue;
+    font-size: 45px;
+    color: #9fb1fe;
+    text-align: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    font-weight: bold;
+}
+
+.ad_info {
+    display: flex;
+    justify-content: space-between;
+}
+
+.pic_container {
+    direction: block;
+    width: 200px;
+    height: 200px;
+    border: 1px solid #9fb1fe;
+    margin-left: 30px;
+}
+
+.title_buttons {
+    display: flex;
+    justify-content: center;
+}
+
+.delete {
+    margin-right: 40px;
+}
+
+.description_container {
+    display: block;
+    height: 80px;
+    width: 300px;
+    text-align: center;
+    color: #9fb1fe;
+    font-family: Helvetica;
+    font-size: 23px;  
+}
+
+.more {
+    color: #957af5;
+    font-size: 24px;
+}
+
+.discount_container {
+    text-align: center;
+    font-family: Epilogue;
+    font-weight: bold;
+    font-size: 36px;
+    color: #7c95ff;
+    margin-top: 5px;
+    margin-right: 20px;
+}
+
+.discount {
+    font-size: 100px;
+    color: #c67cff;
+    margin: 10px;
+}
+
+.hashtags {
+    margin-top: 20px;
+    margin-left: 20px;
+    margin-bottom: 10px;
+    font-family: Epilogue;
+    font-weight: bold;
+    font-size: 18px;
+    color: #687bff;
+}
+
+.reviews {
+    display: block;
+    border-top: 2px solid #9fb1fe;
+    border-bottom: 2px solid #9fb1fe;
+    text-align: center;
+    font-family: Epilogue;
+    font-weight: bold;
+    font-size: 48px;
+    color: #7c95ff;
+    height: 500px;
+}
+
+.review {
+    display: block;
+    height: 75px;
+    border-top: 2px solid #9fb1fe;
+    color: #9fb1fe;
+    font-size: 23px;
+    font-family: Helvetica;
+    text-align: left; 
+}
+
+.no_review {
+    display: block;
+    height: 75px;
+    border-top: 2px solid #9fb1fe;
+    color: #9fb1fe;
+    font-size: 23px;
+    font-family: Helvetica;
```

Comment:
It's better to extract such code duplication, for example:
```
<div class="review-item">Here's a review</div>
<div class="review-item no-review">No review</div>
```

```
.review-item {
    display: block;
    height: 75px;
    border-top: 2px solid #9fb1fe;
    color: #9fb1fe;
    font-size: 23px;
    font-family: Helvetica;
    text-align: left;
}

.review-item.no-review {
    text-align: center; 
}
```

With this approach text-align will be overridden for no-review element.

The same please use across the project.